examples/handeye.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from random import choice
+
+import gym
+
+# noinspection PyUnresolvedReferences
+import gym_handeye
+
+logging.basicConfig(level=logging.DEBUG)
+
+if __name__ == '__main__':
+    hand_eye = gym.make('HandEye3-v0')
+
+    possible_actions = list(range(6))
+
+    for i_episode in range(1):
+        observation = hand_eye.reset()
+
+        for t in range(100):
```

Comment:
Eksperyment kończy się zawsze po 500 krokach - czy to celowe?

Reply:
To środowisko nie ma żadnego określonego celu, po którym można by zakończyć eksperyment, więc ustawiłam 500 kroków, nie jest to jednak niczym uzasadniona liczba.